### PR TITLE
Restore minimal support for Gene Ontology annotations

### PR DESCRIPTION
**Pull request by labkey-jeckels**
#### Rationale
Automatically loading Gene Ontology data has been broken for a long time.

#### Changes
* Switch to HTTP downloads
* Clearly label as deprecated
* Minor code cleanup, including closing of streams and more proactive temp file deletion

### CODE REVIEW

File: ms2/src/org/labkey/ms2/loadGoAutomatic.jsp [hunk 1]
```diff
@@ -43,8 +45,8 @@ LabKey database. If you click "Continue", this server will automatically:
     <li><%=h(loaded ? "Reload" : "Load")%> the database tables with information from the downloaded GO files</li>
 </ul>
 
-Your LabKey Server must be able to connect via the Internet to ftp.geneontology.org. Assuming a reasonably fast
-Internet connection, this process should take less than five minutes to complete. The loading will take place
+Your LabKey Server must be able to connect via the Internet to geneontology.org. This process should take less than
+    five minutes to complete. The loading will take place
 in the background and you can continue to use your LabKey Server normally. If you want, you can monitor the
 process by refreshing the status information on the next page.<br><br>
```
Comment (labkey-adam) [inline ms2/src/org/labkey/ms2/loadGoAutomatic.jsp]: Should remove the Windows-only FTP warning section below

File: ms2/src/org/labkey/ms2/protein/tools/GoLoader.java [hunk 2]
```diff
@@ -90,7 +102,7 @@ public static WebPartView getCurrentStatus(String message)
 
     public static GoLoader getFtpLoader()
```
Comment (labkey-adam) [inline ms2/src/org/labkey/ms2/protein/tools/GoLoader.java]: Not a big deal, but change to `getHttpLoader()`?

**Review by labkey-adam — CHANGES_REQUESTED** [review]
Need to remove FTP warning

**Review by labkey-adam — APPROVED** [review]
Thanks for doing this. Feel free to delete `FTPUtil.java` as a part of this PR.

File: ms2/src/org/labkey/ms2/protein/tools/GoLoader.java (hunk 2 above)
Comment (labkey-jeckels) [reply ms2/src/org/labkey/ms2/protein/tools/GoLoader.java]: Changed.

File: ms2/src/org/labkey/ms2/loadGoAutomatic.jsp (hunk 1 above)
Comment (labkey-jeckels) [reply ms2/src/org/labkey/ms2/loadGoAutomatic.jsp]: Done.

**Comment by labkey-jeckels** [comment]
> Thanks for doing this. Feel free to delete `FTPUtil.java` as a part of this PR.

Thanks for the reminder. I meant to do that but forgot. Now deleted in the FB.

**Pull request merged**